scripts/install_macos.sh

```diff
@@ -16,9 +16,8 @@ elan toolchain install stable
 elan default stable
 
 # Install and configure VS Code
-if ! which code > /dev/null; then
+if ! "/Applications/Visual Studio Code.app/Contents/Resources/app/bin/code" --install-extension leanprover.lean4; then
     brew install --cask visual-studio-code
+    # Install the Lean4 VS Code extension
+    code --install-extension leanprover.lean4
```

Comment:
Shouldn't we also provide the full path here?
```suggestion
    "/Applications/Visual Studio Code.app/Contents/Resources/app/bin/code" --install-extension leanprover.lean4
```
Given this I guess we should capture this in a variable above.